### ISSUE TITLE
Can't get ignore-checks to work

### ISSUE THREAD

**Issue by MaseratiPilot** [aws-cloudformation/cfn-python-lint#1482]
**cfn-lint 0.29.5**

I'm trying to use cfn-lint in some automation scripts and would prefer to ignore certain warnings.
Some are as simple as !Sub lines warning that !Sub isn't required since there is no sub going on when there actually is with the !Sub | to insert a CR at the end of each line.  I've worked around this by playing a simple not relative ${AWS::AccountId} in the text file as a comment but it is annoying.

**cfn-lint test-bucket.yaml** 
W2001 Parameter AccountName not used.
test-bucket.yaml:4:3

**cfn-lint -i w test-bucket.yaml**
_This results in the CLI syntax help displaying_

**Sample CloudFormation Template
test-bucket.yaml**
```
AWSTemplateFormatVersion: '2010-09-09'
Description: Inital Account Setup - CloudTrail, Logs Bucket
Parameters:
  AccountName:
    Description: Account Name
    Type: String
    Default: master
Resources:
  Bucket:
    Type: AWS::S3::Bucket
    Properties:
      AccessControl: Private
      BucketEncryption:
        ServerSideEncryptionConfiguration:
          - ServerSideEncryptionByDefault:
              SSEAlgorithm: AES256
      BucketName: !Sub 'test-${AWS::AccountId}'
      PublicAccessBlockConfiguration:
        BlockPublicAcls: true
        BlockPublicPolicy: true
        IgnorePublicAcls: true
        RestrictPublicBuckets: true

```

**Comment by PatMyron** [comment]
Switch the order and capitalize the w:
```shell
cfn-lint test-bucket.yaml -i W
```

**Comment by MaseratiPilot** [comment]
Thanks Pat - changing the order worked - didn't realize the order was the issue.
